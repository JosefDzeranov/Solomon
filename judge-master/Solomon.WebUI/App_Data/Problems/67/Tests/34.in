1000
622292512 98099223 657
665543017 127498180 794
167658979 218618999 18
526644225 108239959 685
890478436 45150684 127
230455901 460494703 618
853172681 119460658 290
715523587 151958681 663
459602150 400921104 998
583042793 30672504 817
414223925 553030829 383
624251551 302630163 304
179739704 481279631 619
406277204 215759112 41
656822594 47715334 981
621265361 333107849 413
240978286 159030593 424
338560307 360308613 637
13411565 393233260 738
791807095 113617355 674
750844126 132841458 868
113415763 229177769 865
634018215 292249204 359
452931426 154268638 473
53698928 406689745 660
299462334 86392115 914
310524591 568977419 69
263674903 337307646 746
530052286 347484152 162
911003926 79906872 566
724059440 35500978 838
797362180 125647217 884
84824889 752269956 732
123002786 428949221 133
694515542 281642558 883
107569990 214649601 334
704504772 48270465 52
978293662 534105 449
712260593 259572358 694
169068583 234774692 531
655567329 328049436 789
933187000 27501119 272
697502316 87840435 476
892885532 6675608 457
722529523 59247541 372
908026139 91641496 988
790618881 168828587 194
773766587 206717809 872
573996457 186388339 535
310312886 150662881 498
744494291 177360560 348
6285279 663641495 67
78433776 495719787 34
951895576 24789474 326
911877036 67826858 4
759440572 20083209 644
392139782 419729618 102
979140310 13780292 674
756768848 30851493 153
941218900 37118532 18
761013864 1927296 749
575283354 205220464 955
983301783 4857067 996
74522075 419510763 210
640454035 53725348 498
950608238 20487636 113
159105049 480376520 940
66260800 716120759 598
133342437 695966258 390
152715599 771890593 202
400227379 195411844 432
529447569 52423368 466
29789127 589396907 845
297092209 673855746 344
547065914 183806719 164
368215078 156943329 658
535577528 39689670 813
107614910 38458165 869
615825675 282993958 580
951106224 21374084 61
74227157 863813504 867
248837291 603696431 151
743712368 148372538 152
36044405 189643593 433
776341870 193703983 207
185108810 621096495 807
683888491 159950897 975
76580542 914653784 939
167002494 813530621 160
182896586 585333369 527
387038409 231768920 874
315466963 303490173 450
812473083 46838141 783
802116394 61905885 965
551274484 386383625 12
404926490 80002129 652
176189494 398353701 301
178841024 47600341 921
312737844 209400364 1000
152334578 87595138 513
561536731 200396833 498
63254309 919242124 669
156355625 172286711 281
848790736 41875289 385
679166087 44742568 12
966720333 532197 356
643877081 27122279 830
638798853 331380827 391
476346473 280234642 578
510973858 442955693 587
351263108 75665617 105
134478583 487895605 1
172914249 140192456 962
449742572 444897605 429
205717145 378072649 963
97612634 258670126 239
284482093 299431476 915
273048967 228853221 740
83726130 49816885 602
760157797 57207297 289
957366552 29987980 946
146883510 528525637 716
109764460 239308443 743
692562274 141793346 702
513790809 436515977 904
815107179 30921399 831
825731099 51049805 994
399345057 420153535 217
534255147 142214146 523
70050794 694508183 129
729126858 265897690 175
471755748 134123642 578
116646099 347750806 755
615877799 379138951 651
269723688 91236125 826
375044438 428217320 182
202278060 159220869 401
250763793 696465604 329
370203163 35593534 384
387096345 183138850 733
322225153 247595131 140
787555921 109052217 618
663400229 161334944 181
578972574 103063504 178
293088593 520523484 442
722829651 275871799 555
484138211 375493841 878
224256436 602845574 404
826644371 126681291 669
871459070 2110549 122
192499787 483656677 328
970938985 4185663 829
558725578 318144960 13
436582186 110280859 556
443500672 203382279 211
959809292 24936915 21
465985164 428363656 297
196757277 188628988 594
149484810 740750830 81
478865326 88664094 556
429112995 78158478 464
228233782 579665186 725
553748040 7772738 151
302332886 505768614 359
562239800 219244008 649
175089136 381417605 780
380542778 409618383 465
146895139 395432389 434
89113008 385012645 171
327549698 173683876 436
226333719 87940217 535
590181409 238433158 716
124471907 4291581 410
369432930 339000633 575
289496230 698070182 994
823020424 69661974 254
292959545 347920087 470
225028255 155642568 598
665897239 88221784 791
41847929 761312497 597
667711219 38850178 567
137881634 784769680 271
462971877 221405821 910
200476758 420386953 431
824414534 144126049 664
16897112 57510579 77
32329276 105539119 75
579327600 132168420 185
565744891 368624929 377
738475038 251113022 917
799186336 106655600 837
669540498 98677953 582
592641437 391775819 680
4332940 87293821 430
13541039 347206346 173
635961557 118273968 895
329720856 60900575 554
393478828 30931735 273
739314032 191643064 656
63362420 709318119 392
337312793 110059359 925
764877587 16983309 257
896965301 70897860 304
805142972 57199483 525
675956743 21926479 585
462387336 40638829 160
276737055 340671203 208
316065242 296561497 771
900504360 25770343 333
741399049 2100916 866
163023557 313638476 308
715241947 124526414 535
617609792 57542232 477
463758060 514942542 367
691179762 163037499 571
293795786 305540803 677
8172442 144067325 465
364748212 35825290 638
273598597 300812224 649
892359406 104944826 163
582834676 319408935 215
129617939 765703247 290
480146999 194463367 249
640730241 102796805 754
404664379 300940709 66
253730561 385723281 935
760834182 238805736 384
252634490 542183520 90
875038059 16338064 820
987698203 11783782 675
623133546 289045068 509
95838389 119849968 722
365764799 103640822 153
232291397 242752253 888
861356187 26595611 834
380566637 461471638 799
511034826 219864871 387
873587245 82947741 658
257213974 438359440 350
864477137 100075514 591
140155343 283346257 700
839607426 81980976 598
302510314 89732362 642
457850143 96269546 840
363976113 528146360 279
98612388 72593767 885
348383348 326391008 329
351265060 190664802 187
370807229 334901284 550
312760284 215524609 164
259009266 708202497 373
405389974 209354366 942
195034071 2279499 39
582332710 223318931 979
183971854 678323909 616
604167736 11072336 872
374907417 296893427 684
86237987 601864912 497
842708836 83654127 364
235164977 366482655 598
902225973 49528359 446
890768513 78523186 67
937403653 41221997 363
2047201 319980673 358
212088522 123947080 788
658363844 66144662 51
566062160 179536800 395
987319185 9017845 342
425252065 262292383 747
347254294 546136297 361
655633476 55830470 476
424428058 38418366 497
846849024 68862051 524
551891450 69977562 440
458331517 72525869 557
127459521 684801784 35
66205163 751646687 645
516618684 34377626 854
930278049 52196842 185
546055526 244863237 39
763220240 23335878 927
461393004 64735882 966
331605745 85487449 398
336656326 512143544 402
10288485 47260858 534
852865507 97012024 82
687823184 59096109 184
613433146 180392821 229
406304409 498535291 930
438767441 551572342 29
908298485 39209381 804
277109728 598619336 907
130343813 357628914 874
508255024 199438307 516
478615351 180034931 510
210912106 13494319 849
144283097 7818138 963
547182820 369432419 795
261262110 311481554 183
179995612 91986026 887
434296123 223483563 572
10458381 790271014 301
67116902 726649653 689
637653387 333200764 779
556610268 120132898 815
9689959 968161456 927
952327709 9141191 179
502690288 132986273 90
131491580 803775136 920
930042702 739828 156
226624838 578551188 355
474525095 312397623 650
614169785 9707842 155
923054592 42806870 270
671920898 55155908 485
601492826 224214497 65
845063166 45682869 205
912293585 23064539 419
245735143 447613110 124
332425625 408051014 132
774040468 38320223 256
188624605 425537983 623
967742462 19009706 665
146444541 408260252 684
636163065 119008943 126
172143333 166050671 971
632689452 2047295 20
728135122 14344369 508
499956155 449553474 296
828299858 41711027 315
675835761 46219887 612
836368138 16047338 219
748238009 91452178 908
28376786 210396939 877
905995135 34595247 632
926642227 13101059 219
74091573 205302406 898
518608114 419628284 845
934883467 10812011 82
807483826 62134826 380
255981545 168920031 857
683214421 128039438 975
770358574 171097511 669
257960162 352909191 207
675953831 284618768 551
566881497 166436034 537
859884668 81231308 831
956169969 26716405 809
236631808 397865835 577
497599002 476477558 508
420920196 45727460 482
511985097 328845658 426
238690813 689609117 703
120589929 668859473 818
343187869 588159172 574
213901722 554083082 867
491494482 444102550 556
612669444 661886 274
670511719 238801866 18
424953351 273845581 948
589512875 295001631 153
467208106 306013097 132
183871584 512996816 318
6141322 370651807 951
193122449 650423131 32
540956683 248066842 391
514667599 7835077 419
374560261 547674680 589
426828107 535235990 811
618655544 236694380 84
895880826 6082008 251
389838589 29490926 757
282616473 654595333 564
828592636 115498563 914
508546346 379094903 707
833696982 58278216 459
62230756 452569632 719
548012494 353203085 63
329994306 245633827 74
638616235 224449544 761
570642441 276344678 398
44504128 64290315 820
106419347 839402271 564
677795659 130011759 97
765207207 132922839 319
967729926 3512828 282
111132994 800003332 14
135863479 109002489 859
10991617 301405745 728
262434334 433551419 570
834784198 27584019 467
716517640 193192831 599
2979977 23399779 127
71759199 323556594 652
155378568 447734651 47
483832808 377365331 836
327562142 634468607 891
730759993 157984412 563
814389603 145948870 798
671584307 15522926 955
996269568 2711930 666
247671191 533575302 679
595652636 53999056 397
966709095 2141442 164
38446443 439847156 9
688561726 171955233 425
105473072 94469009 199
893107349 19605372 877
267202499 423941479 23
435716321 138399975 567
988295744 3164865 19
455229244 114102699 941
720949318 160409178 503
725541432 7183290 747
671208643 87934909 794
475785612 523134435 120
582773015 368634565 386
992036071 3537827 316
362925730 573951865 106
641628678 315211976 425
25532494 430047556 227
920156894 46544645 152
732723328 163611851 9
205004161 346492471 453
366120736 411902761 805
785284738 12718602 853
109610599 466389687 972
502288274 379133855 467
502086932 494479527 866
985180803 8491502 181
463570148 286852241 259
89119741 373011060 223
126743486 833994183 297
215263171 458061435 354
91526344 550975661 866
296245787 579742802 347
531092466 139136007 851
509998007 125253025 288
859988522 20811381 812
900797276 62227414 200
833243147 164617880 660
345387080 128174789 748
507939069 31721974 615
757525158 32084415 74
75950465 606585170 928
93645078 612917763 603
320022450 86550439 310
795639677 152324963 850
868591978 27733711 140
696927448 243398433 88
694895154 129314158 10
909556796 27419783 491
484493996 46054052 477
739625752 239463133 774
470151992 501303647 214
981262198 13518861 959
828390181 39253292 282
268284055 85674252 703
432935888 480689139 131
219174635 712549385 521
873511098 80309932 905
380055411 397434838 985
941723455 40875506 82
847132364 107001233 34
109535466 445536862 848
336388421 56919207 358
852263037 110283741 210
884156545 28914785 485
51877683 875100770 703
725100144 184868143 851
39220046 149121536 795
236512257 252158759 248
342410594 420158194 171
696403665 288088815 948
71246975 261426609 124
459865107 406623296 352
674651522 132448047 689
705637243 189845186 250
104345707 462953186 419
444697475 93465254 838
929604059 55584249 318
803023769 98800023 791
737541346 161118925 22
214294127 508802800 527
359083491 494599674 572
249509420 613011325 36
709529175 214583337 610
61770304 788273215 895
568071587 269756746 643
660024117 179959340 399
563605614 105552012 224
355256698 47217235 709
249847909 340031902 759
420272616 268459275 67
904829846 37000717 378
911865211 38824069 171
916332024 20463496 464
239764290 206817477 804
186119638 774063862 243
228186756 654873424 758
389910149 390488833 930
534114899 406761398 743
92535163 668184897 406
392803701 226716827 281
631432403 327545516 607
600128239 122287402 859
553155529 417123937 473
140743069 161944065 183
900206724 55853456 563
995794710 3420365 627
377983818 535105951 73
984264458 14004381 17
24713257 77775314 178
266108708 28225816 300
212380740 729605387 537
746059666 236928986 274
304053805 320027678 899
504213072 93209340 930
695993459 90331291 909
209624198 443595514 635
123924964 557905708 175
536860123 86868089 474
408370008 518299984 270
918446603 35694752 746
904896550 88868121 189
893867950 532501 844
471658571 511975399 344
606170400 337707063 690
141118549 44343599 873
897931802 3789837 898
219803735 737002641 297
17084535 70838892 258
358585709 620523339 461
468185109 494027339 129
952481510 20899297 368
296646848 283077620 17
25256463 140604757 270
446799295 532659958 313
823802699 61713435 371
576056156 105899622 620
676260044 207666218 241
743307786 214052038 34
164708370 539369388 633
65417798 440154191 265
984743063 13300619 875
61241997 9465937 841
106849810 504238603 129
615209930 151481386 857
198210856 213828308 655
903690220 91501314 139
636611193 261298073 216
945006163 50779838 629
279559324 283822329 661
599070407 352719912 836
277824377 503550369 328
754925021 134688869 734
690814988 147925423 462
446046829 313384629 748
17865424 889968855 19
601981757 46788265 534
140839921 43547423 50
401162434 93286980 559
852027783 138265732 546
823339308 160030008 256
762023518 44670676 897
848510854 100432005 169
337387892 428745079 647
644990759 68816510 997
81368379 6780289 773
128041070 647391427 850
917008714 12893281 347
192856064 408120617 22
241644677 167446755 275
170590740 32043364 311
769052475 55877931 634
232745971 137199022 749
600883218 380052836 118
489445357 194359836 995
726510705 52754072 276
626071943 85602196 821
610544477 153374477 773
116322433 471631486 632
867704291 24275397 848
77206434 846481428 321
190449500 500557857 167
174184818 478785271 816
103044150 482784625 676
864935680 59238657 252
834792537 36139430 297
708419840 229116374 9
419388807 212673403 951
38248493 187300417 385
334600954 234503238 194
778969732 60282896 919
570433759 344282134 932
226110530 426318582 445
933954688 64364596 738
484737861 368920849 753
460934558 90490096 587
149090215 450680016 152
887258957 41402498 938
459677611 458517492 305
231867807 131072449 920
904389341 26746632 11
521903855 391501345 517
575562187 216532308 236
419514539 40746245 476
126354067 725259887 574
756558271 82582038 13
352958320 412124301 99
828993765 39849673 25
825114707 42568922 773
458077266 26387413 799
119780325 251762871 155
248649715 113947301 941
844856690 59640173 383
900472486 97285930 609
884993529 96074773 330
261433841 109435817 758
610466600 46760903 889
387059022 114087074 36
407959146 538538519 239
743547836 9053293 444
605702953 199696434 57
380403057 191013830 82
222377899 611389603 553
402391647 90717239 937
415599972 570831420 776
702270911 244436923 809
519246116 143029884 226
878386589 44013423 897
786791305 134532672 990
267199419 496748713 590
691341354 240945265 59
70978967 177600029 618
829984155 135011881 69
303869360 554969296 464
827207733 159930896 528
26567287 755814130 4
860440962 107732878 125
870614837 41441649 918
208948444 326796184 444
348750592 352600633 597
136487810 789018222 68
723930243 153500118 723
640505977 202235939 946
113573614 800489377 635
927346549 59706558 220
795307080 36235901 86
382286762 481893791 40
630199894 274577668 417
574754765 64573701 628
7286757 574807091 4
260629630 214565308 562
868484848 2501524 503
597338589 112419346 930
643774564 19182828 807
176998608 26712515 41
790517190 209009393 294
672663994 274657027 161
374848263 55970850 414
689979951 256029597 640
469969568 370309165 829
464204239 395028393 219
164480682 801856998 703
297734634 406249612 563
776767529 89801081 582
577487399 69922226 827
455378949 351166268 952
178470668 69697220 331
48382057 886940319 169
846215179 131320078 885
707824281 142879335 773
26371967 258028804 19
745779771 79651565 70
435030140 396428677 84
458315907 498770717 419
890064567 20576 527
750296050 159080473 819
777712321 3690184 202
128762537 518076098 864
139414065 381599577 718
626157619 239250229 994
564636593 82587072 821
102268976 509259047 741
364771007 392315832 460
474208334 238333135 611
158443362 328056471 513
575039432 208332874 560
276054559 655636755 35
168213087 424595339 688
734734053 173332569 436
542604053 431217971 256
703214043 93941327 964
477489128 356455873 903
37169646 155635250 849
725305505 257234686 851
368337003 330413855 907
681802413 149264677 721
801957803 60782911 291
525168313 444273784 967
595456561 96245425 102
285475772 588795142 982
150973066 396874018 681
327305229 339891303 168
286466313 544813932 196
100252308 599044930 647
227366343 409723344 557
321441532 540704146 363
629061524 73304762 117
185849046 170115380 813
985960875 4437637 221
125360692 507608651 926
214761207 360603355 450
265740306 602769244 137
330665383 424101921 200
901181384 94594827 285
912494445 10613141 120
609634190 43625027 211
621599069 224026429 196
591761648 341129967 442
538480166 274931616 780
202199224 692109830 966
355461733 228124457 384
770684094 161374626 738
756718533 167192608 936
776479164 216421658 614
583886806 261028931 277
252634067 386664875 406
758859378 139835964 928
695582461 190203285 915
20256839 135217021 449
937212530 41799561 531
666869822 3772756 854
816131125 124374100 760
670646189 46778475 499
994485805 5213610 670
670115046 198232719 835
196911058 21826457 139
611757508 316169909 507
325825023 254893863 814
546510950 130333598 426
260289634 380044867 247
860624627 97688196 616
563753668 227405222 105
582392621 114162336 649
703161346 170300765 407
565703575 247282733 408
322293027 612923220 362
55840627 564525141 151
36649278 941119310 121
229011392 132226523 582
957348673 15848534 910
429603437 431735376 328
869027361 33739065 776
998643563 6990 59
135063585 70306471 25
283634934 533397489 192
181766785 328884889 231
515066218 339465560 825
900985382 36367891 407
288233633 578693538 147
220208671 753194589 159
379138237 166316442 221
42960241 174219662 831
308926452 479942375 496
239475218 320140754 679
330381205 642826032 921
207182239 653047490 383
687977110 7590255 8
806815682 51268845 59
620612667 294793955 197
319751760 623494163 820
478944241 294123640 290
418374469 121683470 830
352032657 44191809 994
742861036 155637701 808
813177226 9961192 495
829332531 137898417 126
974969846 14411026 806
913779281 85386787 570
699087078 154442451 340
432249323 474410169 244
376311186 33977168 182
938522381 32025099 833
313272416 634222467 727
943125146 42790430 271
760225048 194361523 589
623378159 160362836 706
561955604 172322313 140
206559172 715646199 531
788331825 141958354 278
94859062 398098355 659
22719266 465336341 895
129998796 449672026 294
800660781 87041987 371
647777109 267310258 715
664018288 29808943 774
896356260 60281662 510
838269607 108455285 243
280741487 335921945 672
108916970 745873605 535
325190497 314484554 35
388048280 484798312 732
55060395 575582806 811
911655568 53806701 499
512412383 8489233 941
914881867 1794577 327
93022565 789476111 994
390006853 189052270 402
856060817 60885586 214
602141704 195154443 961
607304378 236491179 497
208491744 682385062 101
530497211 259404342 984
851924152 115186868 985
529205060 345845430 440
658253343 138003055 26
769788600 211677163 595
718325067 166136518 798
293225801 496520545 672
903390883 38414054 353
505594341 480571883 567
671667458 115243702 172
62285772 872890628 356
377021504 99547462 314
220587783 719181298 911
343838225 476620929 948
292828930 392552232 216
975011732 821563 353
439967011 403511059 387
161289595 367611023 833
526782985 394370549 265
851042231 76228872 503
12248817 540548254 42
324085197 540837864 984
264220596 345593386 520
954453908 2039112 789
3984008 79294020 571
167772274 246645389 265
855436349 119503140 535
640446707 314186579 377
968788726 20331818 695
905101796 51168351 598
293476695 411473695 42
139861941 429410940 368
386215184 522610242 704
233304075 79196721 763
346334414 652254385 481
603251910 185677016 801
189514804 801720926 529
451906828 464993354 71
957487689 41523749 940
285761150 587486514 26
163930358 561539658 108
103447784 109815668 566
188207678 408375651 2
231678787 642578244 614
978859929 17118623 165
528741224 163531828 523
828482911 128021021 32
841439480 125421766 479
33744299 569858248 739
154148473 498108454 383
318785440 368265047 197
976335919 12068647 668
979121005 1562608 500
376303405 355615505 856
326432704 557879541 569
742598879 204833636 766
300923293 44052189 98
883521497 36860475 812
984104760 13250408 163
645679617 92502505 671
837286247 34348697 758
650499285 301361064 303
608526377 373749174 263
213621449 447267449 300
601993289 30341041 914
739579040 228003069 543
385490140 421825311 887
674823174 226447399 384
104885630 68345358 779
391916887 383273557 753
376975693 416301589 432
701627689 6377184 995
115631850 648217324 423
88363494 279085510 608
685042572 283151358 68
237361681 93268289 380
372700894 172505970 924
582196288 357767226 866
835889159 59509146 505
634466337 295970911 451
442215050 304468169 92
874561957 2299305 727
418028481 104095760 201
856963559 40062829 304
609358120 383773360 646
73763557 664758621 586
699786458 181816762 59
271022474 413101997 811
624557737 28772468 506
842218141 33655253 819
90453430 455557050 824
824962693 103897677 938
478665855 298671316 678
444416870 465958957 281
827735862 66738765 438
573758794 85828747 713
986113120 12167171 129
579420372 266453854 543
535382348 219460263 524
860817185 47970376 44
30237315 456912600 511
293755733 419986656 23
675208086 19387805 816
777024383 65984554 104
334338699 387148972 556
413994661 277680105 351
502972418 359132180 876
10669690 5572044 252
977676990 13135480 27
77546895 800448124 810
229464804 320120931 81
941888616 26840373 828
263820132 566898066 948
728638833 229354643 660
65506618 876768027 292
421970679 481676191 82
643578622 197863060 586
861922044 11164882 101
466702897 316962189 164
920889857 34386254 849
589302350 255262835 135
321055841 223043717 319
357450774 192185877 705
322283437 203998922 220
260966088 56272913 558
149960218 687979396 608
799306729 58086478 550
151443864 187965422 857
860548877 65757235 965
746638180 43559499 815
874102909 42872805 20
604092581 69661467 227
371247265 539202901 520
316385581 398634597 710
740154753 196820988 564
310945951 26455891 908
738131877 130762391 343
380123122 500676584 911
614222691 343067561 389
753573890 96605677 22
666625615 233560036 52
210024669 360002917 2
765936027 4146565 628
993264922 5957271 176
298938708 87269867 800
264067823 689666854 737
527290124 156122081 819
705125481 107235728 929
871842704 73931127 842
333952424 117933815 865
82622671 446891738 213
450050972 269817934 271
715144236 219870613 394
134845334 135932697 834
777528152 184468051 645
350546812 306196041 125
440333780 28305962 333
54448567 361990568 92
418727149 432894987 615
265107109 38894956 627
639339193 238863257 289
932568601 1775173 999
637183891 169505427 857
202317523 11858100 627
771061433 90499751 501
411281937 179023310 895
334141193 374678854 591
818224154 100646996 943
129176061 73094966 614
67566868 183241556 783
44509441 551084790 730
28385217 28724029 131
108809152 527320843 487
152878984 792523287 754
501411402 452170142 748
805480429 178214865 417
788697370 5479374 644
379301468 449530047 70
290158415 453429551 589
726772250 4920122 141
255516237 63648798 957
895678280 20785208 937
518549724 68166333 554
522966112 427985374 857
369448005 474288775 361
255404715 362187789 807
