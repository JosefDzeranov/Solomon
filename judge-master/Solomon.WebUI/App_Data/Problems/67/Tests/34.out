30799
50
285 546 569 173 297 574 296 974 950 776 627 807 77 10 313 327 657 734 400 770 886 414 210 623 61 957 680 426 778 612 655 150 896 678 526 530 808 571 310 976 339 838 753 830 38 867 63 411 418 401 